runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -79,15 +86,43 @@
 	@Inject
 	private GrandExchangeConfig config;
 
+	private static Map<String, Integer> geLimitInfo;
+
+
 	@Provides
 	GrandExchangeConfig provideConfig(ConfigManager configManager)
 	{
 		return configManager.getConfig(GrandExchangeConfig.class);
 	}
 
+	private Map<String, Integer> loadGeLdimits()
```

Comment:
Is this meant to be named `loadGeLimits`? It looks like an extra `d` slipped in there